pkg/parser/cse_cmd.sh.gtpl

```diff
@@ -1,15 +1,152 @@
-ADMINUSER="{{.LinuxAdminUsername}}"
-PROVISION_OUTPUT="{{.ProvisionOutput}}"
-{{if .CustomCloudConfig.IsCustomCloud}}
-REPO_DEPOT_ENDPOINT="{{.RepoDepotEndpoint}}"
-{{/* need to properly add {{GetInitAKSCustomCloudFilepath}} >> /var/log/azure/cluster-provision.log 2>&1; */}}
+PROVISION_OUTPUT="/var/log/azure/cluster-provision-cse-output.log";
+echo $(date),$(hostname) > ${PROVISION_OUTPUT};
+{{if true}}
```

Comment:
this used to be `{{if ShouldEnableCustomData}}` but we'll be getting rid of this with self-contained.

Reply:
I am not very sure about this. Needs to check with Anuj. I just followed the same logic in the official karpenter. https://github.com/Azure/karpenter-provider-azure/blob/138a91aaf5cf450303b69c1d26b13ee30810d2ce/pkg/providers/imagefamily/bootstrap/cse_cmd.sh.gtpl#L12C1-L12C25

Reply:
The line you linked to is `{{if .IsAKSCustomCloud}}` which is different than `{{if ShouldEnableCustomData}}`. Karpenter doesn't have this line because they don't rely on AB's customdata scripts.

Reply:
Ah I see. Ya it seems to me it's always true but I can't conclude it either.

Reply:
let us keep "ShouldEnableCustomData" for now, we will sync on it. we will need it for the transition period of going from non self to self

Reply:
Okay, added to contract.